appveyor.yml

```diff
@@ -0,0 +1,22 @@
+version: "{build}"
+
+init:
+  - git config --global core.autocrlf input
```

Comment:
Nops. Removed.
Also removed the environment matrix below. The build is successful. https://ci.appveyor.com/project/am11/node-jscs/build/38 with latest commit: ee2bb1dc5cfaf6f46f55a81f84070b20ef6dfbb7. :tada: